content/docs/command-reference/stage/index.md

```diff
@@ -0,0 +1,25 @@
+# stage
+
+A set of commands to add and list <abbr>stages</abbr>:
+[add](/doc/command-reference/stage/add).
+
+## Synopsis
+
+```usage
+usage: dvc stage [-h] [-q | -v] {add,list}  ...
```

Comment:
If I remove the "extra spacing" here, syntax highlighting fails.

```suggestion
usage: dvc stage [-h] [-q | -v] {add,list} ...
```

If I add a spacing anywhere, for example as following, the syntax highlighting works, but not if I remove it.
```suggestion
usage: dvc stage [-h] [-q | -v] {add, list} ...
```

Reply:
Hmmm strange but you can leave this line exactly as printed by `-h` 👍